addons/code-analysis/addon.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Code analysis",
+  "description": "Analyze the code of the Scratch Community Project",
+  "tags": ["analysis", "community", "project"],
+  "version": "1.31.0",
```

Comment:
This needs to be `versionAdded` instead of `version`. Also, I don't think we'll be adding this until at least version 1.32.0.